tests/robustness/traffic/etcd.go

```diff
@@ -81,8 +81,9 @@ var (
 		leaseTTL:     DefaultLeaseTTL,
 		// Please keep the sum of weights equal 100.
 		requests: []random.ChoiceWeight[etcdRequestType]{
-			{Choice: Get, Weight: 50},
-			{Choice: Put, Weight: 50},
+			{Choice: Get, Weight: 45},
+			{Choice: Put, Weight: 45},
+			{Choice: Put, Weight: 10},
```

Comment:
This doesn't look correct

Reply:
Sorry I was half dazed when I was doing this...  Corrected the new Put to StaleGet now.